FirebaseInAppMessaging/Sources/Runtime/FIRIAMActionURLFollower.m

```diff
@@ -175,8 +175,10 @@ - (BOOL)followURLWithContinueUserActivity:(NSURL *)url {
     FIRLogDebug(kFIRLoggerInAppMessaging, @"I-IAM240004",
                 @"App delegate responds to application:continueUserActivity:restorationHandler:."
                  "Simulating action url opening from a web browser.");
+    // Use string literal to ensure compatibility with Xcode 26 and iOS 18
+    NSString *browsingWebType = @"NSUserActivityTypeBrowsingWeb";
     NSUserActivity *userActivity =
-        [[NSUserActivity alloc] initWithActivityType:NSUserActivityTypeBrowsingWeb];
+        [[NSUserActivity alloc] initWithActivityType:browsingWebType];
```

Comment:
I don't think this will work as intended. `NSUserActivityTypeBrowsingWeb` has a string value. I'm assuming it's some reverse DNS value mentioned here (https://developer.apple.com/documentation/foundation/nsuseractivity/init(activitytype:)?language=objc)